LayoutTests/fast/dom/Range/range-commonroot-notfound.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
What if you removed this DOCTYPE. Wouldn't the test case still crash?

Reply:
No, it doesn't crash.